test/e2e/node/pre_stop.go

```diff
@@ -201,28 +199,8 @@ var _ = SIGDescribe("PreStop", func() {
 		err = podClient.Delete(context.TODO(), pod.Name, *metav1.NewDeleteOptions(gracefulTerminationPeriodSeconds))
 		framework.ExpectNoError(err, "failed to delete pod")
 
-		//wait up to graceful termination period seconds
-		time.Sleep(30 * time.Second)
```

Comment:
should the original test be fixed by changing this timeout to 15 second? 30 seconds is on the edge of graceful termination timeout and pod MAY or MAY NOT run at this ponit. After 15 seconds is is still running for sure if preStop hook is being executed.